monitoring/heartbeats/install-plugin.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Install Heartbeats Plugin
+summary: Enabling endpoint instance monitoring by installing the Heartbeats plugin
+reviewed: 2018-01-05
+component: Heartbeats
+versions: 'Heartbeats:*'
+---
+
+To install the Heartbeats plugin into an endpoint add the following to the endpoint configuration:
+
+snippet: HeartbeatsNew_Enable
+
+WARN: When installing the heartbeat plugin, also configure each instance with a deterministic [Host Identifier](/nservicebus/hosting/override-hostid.md). This identifier is used to keep track of which instance is sending heartbeat messages to the ServiceControl instance.
```

Comment:
Also, doesn't this host stuff apply on heartbeats, monitoring and custom checks? All show the host description if I'm not misttaken. Probably this needs to be mentioned in the monitoring index.

Reply:
That's a good point.